### PR TITLE
incubator-kie-issues#1334: kn workflow create should preserve the ind…

### PR DESCRIPTION
**Pull request by treblereel**
…ex.html file provided by Quarkus

https://github.com/apache/incubator-kie-issues/issues/1334

### CODE REVIEW

**Review by ricardozanini — APPROVED** [review]

File: packages/kn-plugin-workflow/pkg/command/quarkus/create.go [hunk 1]
```diff
@@ -114,7 +114,6 @@ func runCreateProject(cfg CreateQuarkusProjectConfig) (err error) {
 		"mvn",
 		fmt.Sprintf("%s:%s:%s:create", cfg.DependenciesVersion.QuarkusPlatformGroupId, metadata.QuarkusMavenPlugin, cfg.DependenciesVersion.QuarkusVersion),
 		"-DprojectGroupId=org.acme",
-		"-DnoCode",
```
Comment (ricardozanini) [inline packages/kn-plugin-workflow/pkg/command/quarkus/create.go:117]: Please add `--no-wrapper` and `--no-dockerfiles`

Comment (treblereel) [reply packages/kn-plugin-workflow/pkg/command/quarkus/create.go:117]: look like there is no such option [quarkus-maven-plugin](https://quarkus.io/guides/quarkus-maven-plugin#quarkuscreate), but we can delete wrappers after project created/transformed.

Comment (treblereel) [reply packages/kn-plugin-workflow/pkg/command/quarkus/create.go:117]: @ricardozanini As for the Dockerfile, it doesn't seem to be created.
```
new-project % tree
.
├── README.md
├── mvnw
├── mvnw.cmd
├── pom.xml
└── src
    ├── main
    │   ├── docker
    │   │   ├── Dockerfile.jvm
    │   │   ├── Dockerfile.legacy-jar
    │   │   ├── Dockerfile.native
    │   │   └── Dockerfile.native-micro
    │   ├── java
    │   │   └── org
    │   │       └── acme
    │   │           ├── GreetingResource.java
    │   │           └── MyLivenessCheck.java
    │   └── resources
    │       ├── META-INF
    │       │   └── resources
    │       │       └── index.html
    │       ├── application.properties
    │       └── workflow.sw.json
    └── test
        └── java
            └── org
                └── acme
                    ├── GreetingResourceIT.java
                    └── GreetingResourceTest.java
```

Comment (ricardozanini) [reply packages/kn-plugin-workflow/pkg/command/quarkus/create.go:117]: @treblereel weird, the CLI has. Nevertheless, delete all the `docker` directory and `mvn*` files then. 👍

File: packages/kn-plugin-workflow/e2e-tests/quarkus_create_test.go [hunk 2]
```diff
@@ -127,19 +126,13 @@ func RunQuarkusCreateTest(t *testing.T, test CfgTestInputQuarkusCreate) string {
 	expectedFiles := []string{
 		"src/main/resources/application.properties",
 		"src/main/resources/workflow.sw.json",
-		"src/main/docker/Dockerfile.legacy-jar",
-		"src/main/docker/Dockerfile.jvm",
-		"src/main/docker/Dockerfile.native",
-		"src/main/docker/Dockerfile.native-micro",
 		".mvn/wrapper/.gitignore",
 		".mvn/wrapper/MavenWrapperDownloader.java",
 		".mvn/wrapper/maven-wrapper.properties",
```
Comment (ricardozanini) [inline packages/kn-plugin-workflow/e2e-tests/quarkus_create_test.go]: I guess `.mvn` it's not needed also

**Review by ricardozanini — APPROVED** [review]

**Review by domhanak — APPROVED** [review]

**Pull request merged**